### DISCUSSION TITLE
Lightroom Classic

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#66] [General]
# Lightroom Classic

Check if Lightroom Classic is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/18a5c9f2-237e-4cfa-8007-1a61c03fe716

**Comment by DogeBoi3544** [comment]
Note: LrC emulation only is technically supported in the United States. If you want to run it outside of the US, copy all of the LrC files from an x86-64 laptop, then place this into the C drive on your ARM laptop. From there, just open and run (i think there's a run file in the folder)

It won't receive updates or anything like that from Adobe CC, but at least it works.
